django/applications/catmaid/control/annotationadmin.py

```diff
@@ -242,7 +244,7 @@ def copy_annotations(source_pid, target_pid, import_treenodes=True,
             SELECT
             FROM connector_treenode ct
             WHERE ct.project_id=%s
-            ''' % (target_pid, source_pid))
+            ''' % (target_pid, source_pid)) # FIXME "Not all arguments converted during string formatting"
```

Comment:
It looks like the whole SQL call isn't implemented properly. I will need to look at this in more detail.